documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -500,6 +500,86 @@ The generated code of the `updateCarFromDto()` method will update the passed `Ca
 
 Collection- or map-typed properties of the target bean to be updated will be cleared and then populated with the values from the corresponding source collection or map.
 
+[[direct-field-mappings]]
+=== Mappings with direct field access
+
+MapStruct also supports mappings of `public` fields that have no getters/setters. MapStruct will
+use the fields as read/write accessor if it cannot find suitable getter/setter methods for the property.
+
+A field is considered as a read accessor if it is `public` or `public final`. If a field is `static` it is not
+considered as a read accessors.
```

Comment:
We missed that one. Should I just push to master with this change, or create a new PR? This PR is already merged 😄